### PR TITLE
survey.onGetPanelNumber event fix #9227

### LINKED ISSUE #9227: survey.onGetPanelNumber event
We have `onGetQuestionNumber` & `onGetPageNumber` but there is no `onGetPanelNumber`.

### PR DESCRIPTION
**Pull request by andrewtelnov**

### CODE REVIEW

**Review by tsv2013 — APPROVED** [review]

File: packages/survey-core/tests/paneltests.ts
Comment (Darkninja027) [inline packages/survey-core/tests/paneltests.ts:1]: this still doesnt work how it is really intended if you set panel 3s indexing type to 2.1 then q5 has a visual index of 3.1 not 2.1

Comment (Darkninja027) [reply packages/survey-core/tests/paneltests.ts:1]: image

heres what i mean panel three has an index type of 2.1 but the first question starts at 3

**Comment by andrewtelnov** [comment]
@Darkninja027 I going to work on [this issue](https://github.com/surveyjs/survey-library/issues/9186) soon. I agree there are scenarios where SurveyJS works incorrectly.

Thank you,
Andrew

**Comment by Darkninja027** [comment]
@andrewtelnov I appreciate you getting back to me about this, do you have an eta regarding this?

**Pull request merged**